pkg/client/unversioned/client.go

```diff
@@ -97,6 +98,10 @@ func (c *Client) Secrets(namespace string) SecretsInterface {
 	return newSecrets(c, namespace)
 }
 
+func (c *Client) ConfigDatas(namespace string) ConfigDatasInterface {
```

Comment:
For the record, I don't like it because 'data' is a plural.  Thus 'datas' sounds ungrammatical and wrong to me.